packages/@aws-cdk/aws-appconfig-alpha/test/configuration.test.ts

```diff
@@ -149,6 +149,121 @@ describe('configuration', () => {
     Template.fromStack(stack).resourceCountIs('AWS::AppConfig::Deployment', 1);
   });
 
+  test('configuration using deploy method and no environment associated', () => {
+    const stack = new cdk.Stack();
+    const app = new Application(stack, 'MyAppConfig', {
+      name: 'MyApplication',
+    });
+    app.addEnvironment('MyEnv1');
```

Comment:
```suggestion
```

> no environment associated

Shouldn't this be removed?

Reply:
Those are being attached to the application and not the configuration :)